packages/docs/stories/components/checkbox.stories.ts

```diff
@@ -17,88 +16,98 @@ const generateStoryDescription = (attributeName: string) => {
+    { name: 'default', type: 'slot', value: 'Checkbox' },
   ]),
   argTypes,
-  title: 'Components/syn-checkbox',
   parameters: {
     docs: {
       description: generateStoryDescription('default'),
-    }
-  }
+    },
+  },
+  title: 'Components/syn-checkbox',
 };
 export default meta;
 
 type Story = StoryObj;
 
 export const Default = {
-  render: (args: any) => {
-    return generateTemplate({ args });
-  },
   parameters: {
     docs: {
       description: generateStoryDescription('default'),
-    }
-  }
+    },
+  },
+  render: (args: any) => generateTemplate({ args }),
 } as Story;
 
 export const Checked: Story = {
-  render: () => html`<syn-checkbox checked>Checked</syn-checkbox>`,
   parameters: {
     docs: {
       description: generateStoryDescription('checked'),
-    }
-  }
+    },
+  },
+  render: () => html`<syn-checkbox checked>Checked</syn-checkbox>`,
 };
 
 export const Indeterminate: Story = {
-  render: () => html`<syn-checkbox indeterminate>Indeterminate</syn-checkbox>`,
   parameters: {
     docs: {
       description: generateStoryDescription('indeterminate'),
-    }
-  }
+    },
+  },
+  render: () => html`<syn-checkbox indeterminate>Indeterminate</syn-checkbox>`,
 };
 
 export const Disabled: Story = {
-  render: () => html`<syn-checkbox disabled>Disabled</syn-checkbox>`,
   parameters: {
     docs: {
       description: generateStoryDescription('disabled'),
-    }
-  }
+    },
+  },
+  render: () => html`<syn-checkbox disabled>Disabled</syn-checkbox>`,
 };
 
 export const Sizes: Story = {
+  parameters: {
+    docs: {
+      description: generateStoryDescription('sizes'),
+    },
+  },
   render: () => html`
     <div style="display: flex; flex-direction: column; gap: 1rem;">
       <syn-checkbox size="small">Small</syn-checkbox>
       <syn-checkbox size="medium">Medium</syn-checkbox>
       <syn-checkbox size="large">Large</syn-checkbox>
     </div>
   `,
-  parameters: {
-    docs: {
-      description: generateStoryDescription('sizes'),
-    }
-  }
 };
 
 export const CustomValidity: Story = {
+  parameters: {
+    docs: {
+      description: generateStoryDescription('validity'),
+    },
+  },
   play: async ({ canvasElement }) => {
     try {
-      const button = canvasElement.querySelector('button');
+      const form = canvasElement.querySelector('form');
+      const button = canvasElement.querySelector('syn-button') as SynButton;
+      const checkbox = canvasElement.querySelector('syn-checkbox') as SynCheckbox;
+      const errorMessage = "Don't forget to check me!";
 
-      await waitUntil(() => customElements.whenDefined('syn-checkbox'));
+      await waitUntil(() => Promise.allSettled([
+        customElements.whenDefined('syn-checkbox'),
+        customElements.whenDefined('syn-button'),
+      ]));
 
-      if (button) {
+      if (form && button && checkbox) {
+        checkbox.setCustomValidity(errorMessage);
         await userEvent.click(button);
+        button.click();
```

Comment:
You're doing two clicks here, one via userEvent and one via button.click() – is this needed?

Reply:
Yes, this is intended. The first click will make sure you are able to use Storybooks "Interactions", e.g. replay and stuff. The second one is really doing the work.

Problem is that a) `userEvent.click` does not work on web-components in play functions, but is needed for Storybook to work and b) `Element.click` does what it should, but does not notify Storybook :(

Reply:
Understand. Maybe you could write some inline docs do make that more understandable.

Reply:
Maybe I could also create a shared wrapper function so we don not have to repeat it all over? Semantics would be the original ones from TestingUtils, would look like:

```javascript
const triggerClick = async (elm: HTMLElement, ...args?: unknown[]) => {
  await testUtils.click(elm, ...args);
  elm.click();
}
```

What do you think?

Reply:
Hm. I think that sometimes abstractions could lead to confusion. ;)

Reply:
Right, I also don´t know how much steps we will have where this becomes a problem. Seems testUtils fire events differently and lit is not able to see them as there are missing infos. Regular html elements do not seem to have this problem. Maybe this goes away when updating the tooling. Did not find an issue for this, through :(